### ISSUE TITLE
Save config

### ISSUE THREAD

**Issue by Elitalak** [u-235/edytornc#23]
It seems like the config is saved when i close the app, it would be better if it saves the config when i press the ok button in the dialog!

**Comment by u-235** [comment]
Yes, this is roughly what happens. This behaviour can be inconvenient if the app suddenly crashes.  Or is there more inconvenience?

**Comment by Elitalak** [comment]
Yes that is exactly what happens, i have the app running all day, doing some config changes and then the app or windows crashes or maybe at night windows restarts.
If config saves when OK button is pressed there would be no problem!

**Comment by WD50** [comment]
When you're already looking at the configuration, see why the option to "disable file change monitor" and sometimes "show editor tooltips" are not saved
